core/src/env_platform/db_handler.bal

```diff
@@ -57,9 +57,15 @@ function saveApplication(TreeRemovalForm form) returns boolean|error {
                 }
             ]
     };
+    log:printDebug("Constructed application : " + application.toString());
```

Comment:
Sorry missed this:
```suggestion
    log:printDebug("Constructed application: " + application.toString());
```

Reply:
I will resolved those spacing mistakes next onwards. Thank you.